oximeter/instruments/schema/physical-data-link.toml

```diff
@@ -0,0 +1,90 @@
+[target]
+name = "physical_data_link"
+description = "A physical network link on a compute sled"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "rack_id", "sled_id", "hostname", "serial", "link_name" ] },
+    #{ version = 2, fields = [ "rack_id", "sled_id", "serial", "model", "revision", "link_name" ] },
```

Comment:
Is this leftover from testing, or left intentionally? Maybe add a comment if the latter?

Reply:
That was intentional. I have a check in non-test code that the version number is never greater than 1, so that even though we have code to handle and tests to verify multiple versions, you can't actually _create_ multiple versions right now. I was going to remove that as the very last step.

This is a breadcrumb to show what an update would look like. I'll add a comment to that effect.

Reply:
Added a comment in dca0355f3